staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -587,19 +587,29 @@ func (r *Request) tryThrottle(ctx context.Context) error {
 	err := r.rateLimiter.Wait(ctx)
 
 	latency := time.Since(now)
+
+	message := fmt.Sprintf("Throttling request took %v, request: %s:%s", latency, r.verb, r.URL().String())
```

Comment:
if the problem is that people think that they're getting throttled by priority and fairness, why not separate the ones that are 429 related from everything else and explicitly say, "not priority and fairness" on the non-429s

Reply:
It's not exclusive to priority and fairness, retry can happen due to:
- `verb=GET` and the request returned an error that is retriable.
- server sent a `Retry-After` header in the response, most commonly for 429 or 504

This PR just adds the retry info (if applicable) so whoever is going through the logs has more context. I believe, this can shed some light on:
- was this on a retry?
- if so, what triggered the retry?

Today, when we come across this message, we don't know whether it happened before the first try or it happened on a retry.

Reply:
in this code, you can look for a 429 and our server largely (only?) returns 429s because of p&f.   That message should be different than the rest and the rest should reassure people they aren't because of p&f or you'll never stem the tide.